server/controllers/Project.js

```diff
@@ -16,14 +16,14 @@ exports.imageUpload = (req, res) => {
 };
 
 exports.getProject = (req, res) => {
-  const { id } = req.params;
+  const { projectId } = req.params;
 
-  Project.findOne({ _id: id })
+  Project.findOne({ _id: projectId })
```

Comment:
why id -> projectId? not a big deal, but such an unneeded change in my opinion 😆

Reply:
oh is it cause you updated the route? nvm

Reply:
I changed it because it's a bit more specific and explicit since we have user ids and project ids when handling projects.